src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.logo {
+  position: absolute;
+}
+
+.main-logo {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.sub-logo {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search {
+  width: 100%;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  cursor: pointer;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.main-search {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.sub-search {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
```

Comment:
you should have default search-bar and big/small. Not main and sub at the same time. One of them is just search-bar without big/small/main/sub word.